Deploy/stacks/dynamic/stack-clients/src/main/java/com/cmclinnovations/stack/clients/geoserver/GeoServerClient.java

```diff
@@ -36,6 +38,9 @@ public class GeoServerClient extends ContainerClient {
     private final PostGISEndpointConfig postgreSQLEndpoint;
 
     private static GeoServerClient instance = null;
+    private static final Path SERVING_DIRECTORY = Path.of("/var/geoserver/datadir/www");
+    private static final String STATIC_DATA_DIRECTORY = SERVING_DIRECTORY.resolve("static_data").toString();
+    private static final String ICONS_DIRECTORY = SERVING_DIRECTORY.resolve("icons").toString();
```

Comment:
These can probably stay as ```Path``` objects.
```suggestion
    private static final Path STATIC_DATA_DIRECTORY = SERVING_DIRECTORY.resolve("static_data");
    private static final Path ICONS_DIRECTORY = SERVING_DIRECTORY.resolve("icons");
```